### DISCUSSION TITLE
Format of Restart files

### DISCUSSION THREAD

**Discussion by BastiLoe** [OpenRadioss/OpenRadioss#2384] [Q&A]
Hello all,

is there some documentation (except source code) available for the restart file format? I would like to further understand how data is stored in there to maybe get able to modify some of the data. Thanks.

Best Regards

**Comment by elequiniou** [comment]
Dear BastiLoe, thanks for your interest in OpenRadioss. Unfortunately, there is no documentation about restart file format published yet. May I suggest you to directly contact Marian using the address communitymanager@openradioss.org to get in touch and see how to help you with your development before the internal documentation is completed.
Kind Regards,
Eric.
